staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -240,7 +240,8 @@ func (s *DelegatingAuthenticationOptions) getClientCA() (*ClientCertAuthenticati
 		return nil, err
 	}
 	if incluster == nil {
-		return nil, fmt.Errorf("cluster doesn't provide client-ca-file")
+                glog.Warningf("cluster doesn't provide client-ca-file")
```

Comment:
looks like your indentation has changed here.  might want to double-check that.

Might also want to say `cluster doesn't provide a client-ca file, so client certificate authentication won't work`.

Reply:
I'll fix these in the next upload